crates/turborepo-lib/src/commands/mod.rs

```diff
@@ -30,9 +30,7 @@ pub(crate) mod unlink;
 pub struct CommandBase {
     pub repo_root: AbsoluteSystemPathBuf,
     pub ui: UI,
-    #[cfg(test)]
     pub global_config_path: Option<AbsoluteSystemPathBuf>,
-    #[cfg(test)]
```

Comment:
I don't understand why we're removing the `#[cfg(test)]` here. They should be added back

Reply:
I can make the getters public, but this was to allow for the execution state to know where the auth crate should look for tokens